src/GUI/Proof.purs

```diff
@@ -292,7 +305,10 @@ _symbolInput = Proxy :: Proxy "symbolInput"
 type Slots
   = ( symbolInput :: SI.Slot Int )
 
-proof :: forall query input output m. MonadEffect m => H.Component query input output m
+type Output
```

Comment:
Could you make this an ADT with a single `LatexOutput String` constructor instead, to make it more clear and remove the `GUI.SettingsPage` dependency from the `GUI.Proof`. So that we in `GUI.GUI` have something like

```purescript
HH.slot _proof unit GP.proof unit
  (case _ of GP.LatexOutput s -> ActivateModal $ SP.ExportLatexModal s)
```

Reply:
I'll look into this. Had some trouble getting it to work before. Might have some questions later.

Reply:
Scratch that. You gave a super clear description, and I had no trouble implementing it. Thank you!